Tests/js/views/fields/assets/ez-media-view-tests.js

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-media-view-tests', function (Y) {
+    var viewTest, registerTest,
+        Assert = Y.Assert;
+
+    viewTest = new Y.Test.Case(
+        Y.merge(Y.eZ.Test.FieldViewTestCases, {
+            name: "eZ Media View test",
+
+            setUp: function () {
+                this.templateVariablesCount = 5;
+                this.fieldDefinition = {
+                    fieldType: 'ezmedia',
+                    identifier: 'media',
+                    fieldSettings: {
+                        mediaType: 'TYPE_HTML5_AUDIO'
+                    }
+                };
+                this.field = {
+                    fieldValue: {
+                        autoplay: 'false',
+                        fileName: 'marcus.avi',
+                        fileSize: '42000',
+                        hasController: 'false',
+                        height: '42',
+                        id: 'marcusmiller',
+                        inputUri: 'uieuie',
+                        loop: 'false',
+                        mimeType: 'video/mp4',
+                        path: '',
+                        uri: '/var/marcus.avi',
+                        width: '42',
+                    }
+                };
+                this.isEmpty = false;
+                this.view = new Y.eZ.MediaView({
+                    fieldDefinition: this.fieldDefinition,
+                    field: this.field
+                });
+
+                // phantomjs does not support the video tag and the associated
+                // events so to simulate error events, we let
+                // YUI consider those events as custom events, this way,
+                // Y.Node.fire() can be used.
+                this.origErrorCfg = Y.Node.DOM_EVENTS.error;
+                delete Y.Node.DOM_EVENTS.error;
+            },
+
+            "Test field value": function () {
+                this._testValue(
+                    this.field.fieldValue, this.field.fieldValue, "The value in the template should be the fieldValue"
+                );
+            },
+
+            "Test empty field value": function () {
+                this._testValue(
+                    null, null, "The value in the template should be null"
+                );
+            },
+
+            "Test isEmpty with an empty field value": function () {
+                this._testIsEmpty(
+                    {fieldValue: null}, true,
+                    "null should be considered empty"
+                );
+            },
+
+            "Test isEmpty with a field value": function () {
+                this._testIsEmpty(
+                    this.field, false,
+                    "Value should not be considered empty"
+                );
+            },
+
+            "Test isAudio parameter is true": function () {
```

Comment:
usually the test names describe the feature rather than the technical behavior. So here, "Should detect the audio field" would be better.